source/gui/settingsDialogs.py

```diff
@@ -1112,23 +1112,22 @@ def makeSettings(self, settingsSizer):
 		self.lineNumberCheckBox.SetValue(config.conf["documentFormatting"]["reportLineNumber"])
 		settingsSizer.Add(self.lineNumberCheckBox,border=10,flag=wx.BOTTOM)
 		sizer=wx.BoxSizer(wx.HORIZONTAL)
-		# Translators: This is the label for a combobox in the
+		# Translators: This is the label for a combobox controlling the line indentation in the

```

Comment:
nit: "controlling the line indentation" -> "controlling the reporting of line indentation"